modules/rosapps/applications/explorer-old/notifyhook/CMakeLists.txt

```diff
@@ -9,9 +9,6 @@ list(APPEND SOURCE
     ${CMAKE_CURRENT_BINARY_DIR}/notifyhook.def)
 
 add_library(notifyhook MODULE ${SOURCE})
-
-set_module_type(notifyhook win32dll)
-
+set_module_type(notifyhook win32dll IMAGEBASE default)
```

Comment:
Just for understanding: Which base-address will it get by setting `IMAGEBASE default`?
And how to find that out? (now that it is not longer written in the list explicitly)
I am just asking this specifically because notifyhook was part of the ancient shell (a crucial OS component), and therefore is a little more interesting to spend some thoughts on than for most other rosapps modules.